Source/src/modules/ModuleInput.cpp

```diff
@@ -103,21 +104,38 @@ UpdateStatus Hachiko::ModuleInput::PreUpdate(const float delta)
             mouse_pixels_motion.y = sdl_event.motion.yrel;
 
             mouse_normalized_motion.x = mouse_pixels_motion.x * _window_width_inverse;
-            mouse_normalized_motion.y = mouse_pixels_motion.y* _window_height_inverse;
+            mouse_normalized_motion.y = mouse_pixels_motion.y * _window_height_inverse;
 
             // Get monitor-global pixel position of mouse:
             int mouse_global_x = 0;
             int mouse_global_y = 0;
+            
             SDL_GetGlobalMouseState(&mouse_global_x, &mouse_global_y);
-            mouse_pixel_position.x = static_cast<float>(mouse_global_x);
-            mouse_pixel_position.y = static_cast<float>(mouse_global_y);
+            mouse_global_pixel_position.x = static_cast<float>(mouse_global_x);
+            mouse_global_pixel_position.y = static_cast<float>(mouse_global_y);
 
             // Normalized mouse position of SDL:
             // NOTE: Contrary to the pixel position, normalized position does
             // not get updated by the changes when the mouse is outside the
             // window:
             mouse_normalized_position.x = static_cast<float>(sdl_event.motion.x) * _window_width_inverse;
             mouse_normalized_position.y = static_cast<float>(sdl_event.motion.y) * _window_height_inverse;
+
+            // Get OpenGL position of mouse:
+            int height = 0;
```

Comment:
they dont need to be initialized since GetWindowSize will do

u could even do int window_height, window_width; in a single line. Same for the GetWindowPosition

Reply:
```int window_height, window_width;``` this on a single line is not a good habit for c/c++. Consider ```int * i, j;``` there you have ```i``` as ```int*``` but ```j`` as ```int```. So leaving it like that would encourage any maintainer to change the types at once and can lead to loss of time and effort.